webui/react/src/omnibar/Omnibar.tsx

```diff
@@ -1,37 +1,59 @@
 import OmnibarNpm from 'omnibar';
-import React, { useCallback } from 'react';
+import React, { useCallback, useEffect } from 'react';
 
-import { StoreAction, useStoreDispatch } from 'contexts/Store';
+import { StoreAction, useStore, useStoreDispatch } from 'contexts/Store';
+import handleError from 'ErrorHandler';
 import * as Tree from 'omnibar/tree-extension/index';
 import TreeNode from 'omnibar/tree-extension/TreeNode';
 import { BaseNode } from 'omnibar/tree-extension/types';
 import { isTreeNode } from 'omnibar/tree-extension/utils';
 
 import css from './Omnibar.module.scss';
 
+/*
+Ideally we wouldn't need to access the element like this.
+A potential option is use the value prop in combinatio with encoding the tree path into
+the options returned by the tree extension.
+*/
+const omnibarInput = () => document.querySelector(
+  '#omnibar input[type="text"]',
+) as (HTMLInputElement | null);
+
```

Comment:
**suggestion:** I think the module of `omnibar` should expose the input instead of having the consumer going the round about way to get this.

One option is to use the `forwardRef` in `OmnibarNpm` to provide a reference to the input element.

Another option (this might be better) is to provide the `input` element through a handler prop. For example, in your use case, it looks like the goal is to focus on the `input` element when the omnibar opens. So the prop would be something like:
```
interface OmnibarNpmProps {
  onShow: (input: HTMLInputElement) => void;
  ...
}
```